content/getting-started/README.md

```diff
@@ -15,47 +15,55 @@ The `kubectl` tool uses a `kubeconfig` file to get the information it needs in o
 
 `kubectl` will by default look for a file named `config` in the `$HOME/.kube/` folder. You can also override this by having the absolute path of the file in the environment variable `KUBECONFIG`. 
 
-## ScaleFT
-<todo>
+## Connect to ScaleFT
+
+In order to reach our clusters, you have to be connected to the right ScaleFT host. For GCP, it's one host per cluster (`dev-gcp` and `prod-gcp`), for on-premise you select `devWeb02`.
```

Comment:
Hvordan får man alle clustern i `navTunnel`'en? Jeg har kun `devWeb02`.

Reply:
De kommer dit av seg selv

Reply:
Ok, for prøvde å følge oppskriften, men stopper opp på at jeg ikke får de opp.

Reply:
Prøv igjen nå, du skal ha fått pusha en oppdatering av navtunnel nå

Reply:
Fikk ny oppdatering av navTunnelen, så nå kan jeg velge `connect gcp-dev`. Og med oppdatert kubeconfig så kom jeg inn! Mangler `gcp-ci` da :/

Og er det meningen at alle skal ha tilgang til GCP nå? Siden det vil dukke opp hos folk nå.